src/ray/gcs/tables.cc

```diff
@@ -415,6 +421,7 @@ template class Table<TaskID, ray::protocol::Task>;
 template class Table<TaskID, TaskTableData>;
 template class Log<ActorID, ActorTableData>;
 template class Log<TaskID, TaskReconstructionData>;
+template class Table<ClientID, TaskLeaseData>;
```

Comment:
Can you remind me, when do we use `Table` and when do we use `Log`?

Reply:
We use a `Table` if it's just one entry at a time and a `Log` if there is a series of changes that we want to store (e.g., additions and deletions from a node for objects).